src/functions-and-arrays.js

```diff
@@ -52,15 +109,33 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(array) {
+  if (array.length === 0 ) {
+    return null;
+  } for ( let i = 0; i < array.length ; i++) {
+    if (array.indexOf(array[i]) === array.lastIndexOf(array[i])) {
```

Comment:
Here you should've use another approach. 
Maybe you could check if there's an existing matching element and work over it. 
```
let uniqueArray = [];
  for (let i = 0; i < arr.length; i++) {
    if (!uniqueArray.includes(arr[i])) {
      uniqueArray.push(arr[i]);
    }
  }
  return uniqueArray;
```